runtimes/pythonrt/py.go

```diff
@@ -36,9 +37,40 @@ func createTar(fs fs.FS) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
+type Version struct {
+	Major int
+	Minor int
+}
+
 type exeInfo struct {
 	Exe     string
-	Version string
+	Version Version
+}
+
+func parsePyVersion(s string) (major, minor int, err error) {
```

Comment:
it feels a bit overcomplicated to me..
maybe just scanf?

Reply:
or a regex?